2025-05-31 12:38:33.114 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-05-31 12:38:33.115 | INFO     | __main__:main:38 - Starting database setup...
2025-05-31 12:38:33.116 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-05-31 12:38:33.132 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-05-31 12:38:33.142 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-05-31 12:38:33.152 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql
2025-05-31 12:38:33.152 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-05-31 12:38:33.153 | INFO     | __main__:main:65 - Database connection closed.
2025-05-31 12:38:51.614 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-05-31 12:38:51.614 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-05-31 12:38:51.615 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-05-31 12:38:51.616 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-05-31 12:38:51.629 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-05-31 12:38:51.629 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-05-31 12:38:51.629 | INFO     | __main__:main:63 - Database connection closed.
2025-05-31 13:52:25.591 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-05-31 13:52:25.591 | INFO     | __main__:main:80 - Starting data export...
2025-05-31 13:52:25.592 | INFO     | __main__:main:88 - Created project folder: C:\Repos\datafun-05-sql\project_data
2025-05-31 13:52:25.611 | INFO     | __main__:export_to_csv:74 - Created C:\Repos\datafun-05-sql\project_data\authors.csv
2025-05-31 13:52:25.612 | INFO     | __main__:export_to_csv:74 - Created C:\Repos\datafun-05-sql\project_data\books.csv
2025-05-31 13:52:25.612 | INFO     | __main__:main:97 - Data export completed successfully
2025-05-31 14:54:25.359 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-05-31 14:54:25.359 | INFO     | __main__:main:41 - Starting database setup...
2025-05-31 14:54:25.361 | INFO     | __main__:main:56 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-05-31 14:54:25.364 | INFO     | __main__:execute_sql_file:18 - Executed: C:\Repos\datafun-05-sql\sql_queries\query_aggregations.sql
2025-05-31 14:54:25.364 | INFO     | __main__:execute_sql_file:18 - Executed: C:\Repos\datafun-05-sql\sql_queries\query_filter.sql
2025-05-31 14:54:25.365 | INFO     | __main__:execute_sql_file:18 - Executed: C:\Repos\datafun-05-sql\sql_queries\query_group_by.sql
2025-05-31 14:54:25.365 | INFO     | __main__:execute_sql_file:18 - Executed: C:\Repos\datafun-05-sql\sql_queries\query_joins.sql
2025-05-31 14:54:25.365 | INFO     | __main__:execute_sql_file:18 - Executed: C:\Repos\datafun-05-sql\sql_queries\query_sorting.sql
2025-05-31 14:54:25.365 | INFO     | __main__:main:61 - Database setup completed successfully.
2025-05-31 14:54:25.366 | INFO     | __main__:main:66 - Database connection closed.
2025-05-31 15:08:32.445 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-05-31 15:08:32.446 | INFO     | __main__:main:61 - Starting query processing...
2025-05-31 15:08:32.446 | ERROR    | __main__:process_all_queries:36 - Queries folder not found: datafun-05-sql\sql_queries
2025-05-31 15:08:32.446 | INFO     | __main__:main:71 - Query processing completed
2025-05-31 15:21:30.528 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-05-31 15:21:30.529 | INFO     | __main__:main:52 - Starting query execution...
2025-05-31 15:21:30.529 | INFO     | __main__:main:68 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-05-31 15:21:30.542 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_aggregations.sql → C:\Repos\datafun-05-sql\sql_output\query_aggregations.csv
2025-05-31 15:21:30.545 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_filter.sql → C:\Repos\datafun-05-sql\sql_output\query_filter.csv
2025-05-31 15:21:30.547 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_group_by.sql → C:\Repos\datafun-05-sql\sql_output\query_group_by.csv
2025-05-31 15:21:30.549 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_joins.sql → C:\Repos\datafun-05-sql\sql_output\query_joins.csv
2025-05-31 15:21:30.550 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_sorting.sql → C:\Repos\datafun-05-sql\sql_output\query_sorting.csv
2025-05-31 15:21:30.551 | INFO     | __main__:main:73 - Query execution and saving completed successfully.
2025-05-31 15:21:30.551 | INFO     | __main__:main:78 - Database connection closed.
2025-06-02 16:43:23.777 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 16:43:23.777 | INFO     | __main__:main:38 - Starting database setup...
2025-06-02 16:43:23.779 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 16:43:23.801 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-06-02 16:43:23.820 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-06-02 16:43:23.827 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql: near "'1984'": syntax error
2025-06-02 16:43:23.828 | ERROR    | __main__:main:62 - Error during database setup: near "'1984'": syntax error
2025-06-02 16:43:23.828 | INFO     | __main__:main:65 - Database connection closed.
2025-06-02 16:45:26.363 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 16:45:26.363 | INFO     | __main__:main:38 - Starting database setup...
2025-06-02 16:45:26.365 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 16:45:26.378 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-06-02 16:45:26.387 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-06-02 16:45:26.393 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql: near "'1984'": syntax error
2025-06-02 16:45:26.393 | ERROR    | __main__:main:62 - Error during database setup: near "'1984'": syntax error
2025-06-02 16:45:26.394 | INFO     | __main__:main:65 - Database connection closed.
2025-06-02 16:51:32.863 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 16:51:32.863 | INFO     | __main__:main:38 - Starting database setup...
2025-06-02 16:51:32.865 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 16:51:32.877 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-06-02 16:51:32.885 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-06-02 16:51:32.890 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql: near "'1984'": syntax error
2025-06-02 16:51:32.890 | ERROR    | __main__:main:62 - Error during database setup: near "'1984'": syntax error
2025-06-02 16:51:32.890 | INFO     | __main__:main:65 - Database connection closed.
2025-06-02 16:51:49.377 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 16:51:49.377 | INFO     | __main__:main:38 - Starting database setup...
2025-06-02 16:51:49.379 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 16:51:49.394 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-06-02 16:51:49.404 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-06-02 16:51:49.418 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql: near "INSERT": syntax error
2025-06-02 16:51:49.419 | ERROR    | __main__:main:62 - Error during database setup: near "INSERT": syntax error
2025-06-02 16:51:49.419 | INFO     | __main__:main:65 - Database connection closed.
2025-06-02 16:54:10.395 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 16:54:10.396 | INFO     | __main__:main:38 - Starting database setup...
2025-06-02 16:54:10.398 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 16:54:10.411 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-06-02 16:54:10.422 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-06-02 16:54:10.445 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql
2025-06-02 16:54:10.445 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-06-02 16:54:10.446 | INFO     | __main__:main:65 - Database connection closed.
2025-06-02 16:56:28.754 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 16:56:28.754 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-02 16:56:28.755 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 16:56:28.772 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-02 16:56:28.783 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-02 16:56:28.784 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-02 16:56:28.784 | INFO     | __main__:main:63 - Database connection closed.
2025-06-02 17:07:28.407 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 17:07:28.407 | INFO     | __main__:main:52 - Starting query execution...
2025-06-02 17:07:28.408 | INFO     | __main__:main:68 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 17:07:28.422 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_aggregations.sql → C:\Repos\datafun-05-sql\sql_output\query_aggregations.csv
2025-06-02 17:07:28.424 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_filter.sql → C:\Repos\datafun-05-sql\sql_output\query_filter.csv
2025-06-02 17:07:28.426 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_group_by.sql → C:\Repos\datafun-05-sql\sql_output\query_group_by.csv
2025-06-02 17:07:28.427 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_joins.sql → C:\Repos\datafun-05-sql\sql_output\query_joins.csv
2025-06-02 17:07:28.432 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_select_statement.sql → C:\Repos\datafun-05-sql\sql_output\query_select_statement.csv
2025-06-02 17:07:28.435 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_sorting.sql → C:\Repos\datafun-05-sql\sql_output\query_sorting.csv
2025-06-02 17:07:28.436 | INFO     | __main__:main:73 - Query execution and saving completed successfully.
2025-06-02 17:07:28.436 | INFO     | __main__:main:78 - Database connection closed.
2025-06-02 17:09:39.983 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 17:09:39.984 | INFO     | __main__:main:52 - Starting query execution...
2025-06-02 17:09:39.985 | INFO     | __main__:main:68 - Connected to database: C:\Repos\datafun-05-sql\data\db.sqlite
2025-06-02 17:09:39.989 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_aggregations.sql → C:\Repos\datafun-05-sql\sql_output\query_aggregations.csv
2025-06-02 17:09:39.991 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_filter.sql → C:\Repos\datafun-05-sql\sql_output\query_filter.csv
2025-06-02 17:09:39.992 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_group_by.sql → C:\Repos\datafun-05-sql\sql_output\query_group_by.csv
2025-06-02 17:09:39.994 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_joins.sql → C:\Repos\datafun-05-sql\sql_output\query_joins.csv
2025-06-02 17:09:39.995 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_select_statement.sql → C:\Repos\datafun-05-sql\sql_output\query_select_statement.csv
2025-06-02 17:09:39.998 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_sorting.sql → C:\Repos\datafun-05-sql\sql_output\query_sorting.csv
2025-06-02 17:09:39.998 | INFO     | __main__:main:73 - Query execution and saving completed successfully.
2025-06-02 17:09:39.999 | INFO     | __main__:main:78 - Database connection closed.
2025-06-02 17:29:46.479 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 17:29:46.479 | INFO     | __main__:main:38 - Starting database setup...
2025-06-02 17:29:46.481 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-02 17:29:46.486 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-06-02 17:29:46.499 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-06-02 17:29:46.508 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql
2025-06-02 17:29:46.509 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-06-02 17:29:46.509 | INFO     | __main__:main:65 - Database connection closed.
2025-06-02 17:34:45.896 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 17:34:45.896 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-02 17:34:45.897 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-02 17:34:45.904 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-02 17:34:45.911 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-02 17:34:45.912 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-02 17:34:45.912 | INFO     | __main__:main:63 - Database connection closed.
2025-06-02 17:40:03.561 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-02 17:40:03.561 | INFO     | __main__:main:52 - Starting query execution...
2025-06-02 17:40:03.562 | INFO     | __main__:main:68 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-02 17:40:03.567 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_aggregations.sql → C:\Repos\datafun-05-sql\sql_output\query_aggregations.csv
2025-06-02 17:40:03.569 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_filter.sql → C:\Repos\datafun-05-sql\sql_output\query_filter.csv
2025-06-02 17:40:03.571 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_group_by.sql → C:\Repos\datafun-05-sql\sql_output\query_group_by.csv
2025-06-02 17:40:03.574 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_joins.sql → C:\Repos\datafun-05-sql\sql_output\query_joins.csv
2025-06-02 17:40:03.577 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_select_statement.sql → C:\Repos\datafun-05-sql\sql_output\query_select_statement.csv
2025-06-02 17:40:03.579 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_sorting.sql → C:\Repos\datafun-05-sql\sql_output\query_sorting.csv
2025-06-02 17:40:03.579 | INFO     | __main__:main:73 - Query execution and saving completed successfully.
2025-06-02 17:40:03.580 | INFO     | __main__:main:78 - Database connection closed.
2025-06-06 15:26:14.088 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:26:14.088 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:26:14.091 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:26:14.097 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:26:14.098 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\update_records.sql: duplicate column name: author_age_at_publication
2025-06-06 15:26:14.098 | ERROR    | __main__:main:60 - Error during feature queries execution: duplicate column name: author_age_at_publication
2025-06-06 15:26:14.098 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:26:54.581 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:26:54.581 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:26:54.582 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:26:54.583 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:26:54.600 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:26:54.600 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:26:54.600 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:31:58.683 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:31:58.684 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:31:58.684 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:31:58.685 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:31:58.686 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:31:58.686 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:31:58.687 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:32:36.322 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:32:36.322 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:32:36.323 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:32:36.336 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:32:36.337 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:32:36.337 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:32:36.337 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:33:55.558 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:33:55.559 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:33:55.559 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:33:55.572 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:33:55.573 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:33:55.574 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:33:55.574 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:36:57.367 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:36:57.367 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:36:57.367 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:36:57.386 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:36:57.387 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:36:57.387 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:36:57.388 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:38:36.180 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:38:36.181 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:38:36.181 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:38:36.184 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:38:36.185 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:38:36.185 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:38:36.185 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:39:58.867 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:39:58.868 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:39:58.869 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:39:58.878 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:39:58.879 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:39:58.879 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:39:58.880 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:42:53.783 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:42:53.783 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:42:53.784 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:42:53.785 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near "(": syntax error
2025-06-06 15:42:53.786 | ERROR    | __main__:main:60 - Error during feature queries execution: near "(": syntax error
2025-06-06 15:42:53.786 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:44:11.618 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:44:11.619 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:44:11.619 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:44:11.621 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near "(": syntax error
2025-06-06 15:44:11.621 | ERROR    | __main__:main:60 - Error during feature queries execution: near "(": syntax error
2025-06-06 15:44:11.622 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:44:49.832 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:44:49.833 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:44:49.833 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:44:49.845 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near "(": syntax error
2025-06-06 15:44:49.846 | ERROR    | __main__:main:60 - Error during feature queries execution: near "(": syntax error
2025-06-06 15:44:49.846 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:49:12.412 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:49:12.412 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:49:12.413 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:49:12.427 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:49:12.428 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:49:12.428 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:49:12.428 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:50:16.042 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:50:16.043 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:50:16.044 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:50:16.067 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 15:50:16.067 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 15:50:16.067 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 15:50:16.068 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 15:59:31.102 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 15:59:31.102 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 15:59:31.102 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 15:59:31.116 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near "WHERE": syntax error
2025-06-06 15:59:31.116 | ERROR    | __main__:main:60 - Error during feature queries execution: near "WHERE": syntax error
2025-06-06 15:59:31.116 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 16:01:25.000 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 16:01:25.000 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 16:01:25.001 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 16:01:25.011 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near ";": syntax error
2025-06-06 16:01:25.011 | ERROR    | __main__:main:60 - Error during feature queries execution: near ";": syntax error
2025-06-06 16:01:25.011 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 16:01:51.640 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 16:01:51.640 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 16:01:51.641 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 16:01:51.663 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 16:01:51.664 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\update_records.sql: duplicate column name: author_age_at_publication
2025-06-06 16:01:51.665 | ERROR    | __main__:main:60 - Error during feature queries execution: duplicate column name: author_age_at_publication
2025-06-06 16:01:51.665 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 16:02:23.288 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 16:02:23.289 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 16:02:23.289 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 16:02:23.290 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 16:02:23.296 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 16:02:23.296 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 16:02:23.296 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 20:55:26.148 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 20:55:26.149 | INFO     | __main__:main:38 - Starting database setup...
2025-06-06 20:55:26.150 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 20:55:26.161 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\01_drop_tables.sql
2025-06-06 20:55:26.201 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\02_create_tables.sql
2025-06-06 20:55:26.227 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Repos\datafun-05-sql\sql_create\03_insert_records.sql
2025-06-06 20:55:26.227 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-06-06 20:55:26.227 | INFO     | __main__:main:65 - Database connection closed.
2025-06-06 21:19:18.089 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 21:19:18.090 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 21:19:18.090 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 21:19:18.100 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near "authors": syntax error
2025-06-06 21:19:18.100 | ERROR    | __main__:main:60 - Error during feature queries execution: near "authors": syntax error
2025-06-06 21:19:18.100 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 21:20:28.456 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 21:20:28.457 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 21:20:28.457 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 21:20:28.465 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near "INNER": syntax error
2025-06-06 21:20:28.465 | ERROR    | __main__:main:60 - Error during feature queries execution: near "INNER": syntax error
2025-06-06 21:20:28.466 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 21:23:23.448 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 21:23:23.448 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 21:23:23.449 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 21:23:23.450 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Repos\datafun-05-sql\sql_features\delete_records.sql: near ";": syntax error
2025-06-06 21:23:23.450 | ERROR    | __main__:main:60 - Error during feature queries execution: near ";": syntax error
2025-06-06 21:23:23.451 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 21:24:10.837 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 21:24:10.838 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-06-06 21:24:10.838 | INFO     | __main__:main:52 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 21:24:10.853 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\delete_records.sql
2025-06-06 21:24:10.861 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Repos\datafun-05-sql\sql_features\update_records.sql
2025-06-06 21:24:10.862 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-06-06 21:24:10.862 | INFO     | __main__:main:63 - Database connection closed.
2025-06-06 21:47:21.737 | INFO     | utils_logger:<module>:41 - Logging to file: logs\project_log.log
2025-06-06 21:47:21.738 | INFO     | __main__:main:52 - Starting query execution...
2025-06-06 21:47:21.739 | INFO     | __main__:main:68 - Connected to database: C:\Repos\datafun-05-sql\data\project.sqlite3
2025-06-06 21:47:21.750 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_aggregations.sql → C:\Repos\datafun-05-sql\sql_output\query_aggregations.csv
2025-06-06 21:47:21.752 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_filter.sql → C:\Repos\datafun-05-sql\sql_output\query_filter.csv
2025-06-06 21:47:21.755 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_group_by.sql → C:\Repos\datafun-05-sql\sql_output\query_group_by.csv
2025-06-06 21:47:21.757 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_joins.sql → C:\Repos\datafun-05-sql\sql_output\query_joins.csv
2025-06-06 21:47:21.761 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_select_statement.sql → C:\Repos\datafun-05-sql\sql_output\query_select_statement.csv
2025-06-06 21:47:21.763 | INFO     | __main__:execute_and_save_query:24 - Executed and saved: C:\Repos\datafun-05-sql\sql_queries\query_sorting.sql → C:\Repos\datafun-05-sql\sql_output\query_sorting.csv
2025-06-06 21:47:21.764 | INFO     | __main__:main:73 - Query execution and saving completed successfully.
2025-06-06 21:47:21.764 | INFO     | __main__:main:78 - Database connection closed.
